libmscore/sym.cpp

```diff
@@ -5321,6 +5321,7 @@ QVector<oldName> oldNames = {
       {"32' rest",                              SymId::rest32nd },            // rests.5
       {"64' rest",                              SymId::rest64th },            // rests.6
       {"128' rest",                             SymId::rest128th },           // rests.7
+      {"256' rest",                             SymId::rest256th },           // rests.8
```

Comment:
I agree it is very risky and the tests are structured that it's hard to know if it's really safe. If we're targeting for 3.0 maybe it's better to put this on the backburner for a while and, as another project, move the entire object model off of ticks entirely. I think the whole headache here is really indicative that it would be much better to just represent durations and locations purely in terms of rational fractions. It would make future changes like this much, much easier, and it would also allow us to easily support arbitrarily nested tuples with far less need for sanity checks and relying on rounding errors(!!!).